renderer/nyaovim-app.ts

```diff
@@ -164,6 +164,117 @@ Polymer({
                     client.command(c);
                 }
             });
+
+            ipc.on('nyaovim:copy', (_: Electron.IpcRendererEvent) => {
+                // get current vim mode
+                var m = client.commandOutput('echo mode()');
+                m.then(value => {
+                    //  mode() returns a strange '\n' at the beginning, why?
```

Comment:
That's a spec of Vim.  All vim commands' outputs start with `\n`.  For example, `:echo blah` outputs `\nblah` (unlike other popular scripts, Vim adds newline to the head of output.)